client/src/components/Ticketing/ticketingmanager/showings/InstancesPage.tsx

```diff
@@ -38,14 +38,28 @@ const InstancesPage = () => {
   return (
     <div className='w-full h-screen overflow-x-hidden absolute'>
       <div className='md:ml-[18rem] md:mt-40 sm:mt-[11rem]
-       sm:ml-[5rem] sm:mr-[5rem] sm:mb-[11rem]'>
-        <h1 className='font-bold text-5xl bg-clip-text text-transparent bg-gradient-to-r from-sky-500 to-indigo-500 mb-10 pb-4'>Select Event</h1>
+       sm:ml-[5rem] sm:mr-[5rem] sm:mb-[11rem] h-full'>
+        <div className ={'flex flex-row justify-between w-full mb-14'}>
+          <h1 className='col-span-2 min-[678px]:col-span-1 font-bold text-5xl bg-clip-text
+           text-transparent bg-gradient-to-r from-sky-500
+            to-indigo-500' >Select Event</h1>
+          <button
+            onClick={() => navigate(`/ticketing/showings/v2/0`)}
+            className={'bg-green-600 hover:bg-green-700 h-fit disabled:bg-gray-600 text-white p-3 font-bold rounded-xl flex flex-row'}
+          >
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-6 h-6 my-auto pr-1">
+              <path strokeLinecap="round" strokeLinejoin="round" d="M12 6v12m6-6H6" />
+            </svg>
+
+            Add Event
+          </button>
+        </div>
         <ul className='md:grid md:grid-cols-2 md:gap-8 sm:grid sm:grid-cols-1 sm:gap-4 mt-9'>
           {allEvents.map((event) => (
             <li key={event.id}>
               <button
-                onClick={() => navigate(`/ticketing/showings/${event.id}`)}
-                className="shadow-xl rounded-xl hover:scale-105 transition duration-300 ease-in-out w-full"
+                onClick={() => navigate(`/ticketing/showings/v2/${event.id}`)}
```

Comment:
I know that we're using the v2 API in this case, but since this is a URL route, what do you think about dropping the `v2/` portion of the URL here and in `App.tsx`?